app/src/main/res/layout/activity_main.xml

```diff
@@ -10,12 +11,21 @@
         style="?android:attr/progressBarStyle"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
+        android:visibility="gone"
         android:layout_gravity="top|center" />
 
     <android.support.v7.widget.RecyclerView
         android:id="@+id/recycleView"
         android:layout_width="match_parent"
         android:layout_height="match_parent" />
 
+    <android.support.design.widget.FloatingActionButton
+        android:id="@+id/refreshFloatingButton"
+        android:layout_width="74dp"
```

Comment:
Please create a dimens value for this values

Reply:
done